### ISSUE TITLE
Documentation is missing a proper legend for CAI scales (OSM relations with cai_scale tag)

### ISSUE THREAD

**Issue by Max1234-Ita** [osmandapp/OsmAnd#20598]
### Description

While looking for information about the CAI scale of some Alpine trail, a user pointed me to this tables:

SAC scale:
<img width=600 src="https://github.com/user-attachments/assets/2ff1b1b7-cde2-4c49-8d9f-e0404b76de5c">

CAI scale:
<img width=600 src="https://github.com/user-attachments/assets/44ec192d-72e3-4f23-b746-c29468e3b9f4">

In the second picture, I'm not able to tell the difference between the fourth line style ("For expert hikers with equipment") and the fifth ("Hiking in snowy condition"), since the 2 lines are identical.

Knowing that the picture was taken from the "legend created by a 3-rd party plugin, I went to the Documentation page in osmand.net (https://osmand.net/docs/user/map-legend/osmand#path-difficulty), but found that only the table of SAC scales is present; the CAI scales are just named in the top of the page but the related table is missing.

### Steps to reproduce

1. Open the OsmAnd documentation page: https://osmand.net/docs/user/map-legend/osmand
2. Scroll down to the _Path difficulty_ section.

### Actual result

There's a table for SAC-Scale only, CAI Scale is completely missing.

### Expected result

A legend for CAI Scale should be present in the documentation, showing the path difficulty with levels listed as per OSM Wiki page (https://wiki.openstreetmap.org/wiki/Proposal:Cai_scale) and corresponding line style.

### Your Environment (required)

**WARNING** Crash-Logs **MAY** contain information you deem sensitive.
Review this **CAREFULLY** before posting your issue!

```
OsmAnd Version: Any
Android/iOS version: Any
Device model: Any (problem is in the OsmAnd web site)
Crash-Logs: ? None
```

**Comment by yuriiurshuliak** [comment]
The bug has been reproduced. The documentation includes only the SAC scale table, with no table for the CAI scale. Additionally, the plugin does not show any distinction between the two scales—they both display as the same dotted lines.
https://osmand.net/docs/user/map-legend/osmand/#path-difficulty


OsmAnd~ 4.9.0#2976m, released: 2024-08-14

<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/45758940-4495-45d5-8d24-2eaeccd73fe8"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/user-attachments/assets/dc653f26-b8d4-4c76-b11e-db60a1ffb797"  width="300" height="auto"/> </th>
  </tr>
</table>

**Comment by Max1234-Ita** [comment]
> The bug has been reproduced. The documentation includes only the SAC scale table, with no table for the CAI scale. Additionally, the plugin does not show any distinction between the two scales—they both display as the same dotted lines. https://osmand.net/docs/user/map-legend/osmand/#path-difficulty
> 
> OsmAnd~ 4.9.0#2976m, released: 2024-08-14
> <img alt="" width="300" height="auto" src="https://github.com/user-attachments/assets/45758940-4495-45d5-8d24-2eaeccd73fe8"> 	<img alt="" width="300" height="auto" src="https://github.com/user-attachments/assets/dc653f26-b8d4-4c76-b11e-db60a1ffb797">


<BR>
<BR>
Regarding the line style, it looks like setting the Touring View in OsmAnd results in colors being properly shown (see also [this post from the Italian Telegram Group](https://t.me/itosmand/21992?single) ):
<BR>

<img width=360 src="https://github.com/user-attachments/assets/c3e8314d-d24f-4f16-a826-275d72346af0">

So probably the same style shall be applied also for the Default view and the others.

Thanks for your work!

**Comment by vshcherb** [comment]
We could also put screenshots here https://osmand.net/docs/user/map/routes#hiking-trails-difficulty-grade to not overwhelm legend

**Comment by sonora** [comment]
> We could also put screenshots here https://osmand.net/docs/user/map/routes#hiking-trails-difficulty-grade to not overwhelm legend

In my mind our automted legend mechanism is fantastic, because it also serves as a test for the renderers and any issues they may produce! I would accept it being lengthy for the sake of it being complete. Screenshots would mean separate effort, and run the risk of becoming outdated.

Perhaps we could also add proposed=yes highways to the legend (below construction).

And I would swap the position of "Residential" and "Unclassified" highways, to strictly have things in significance order. 😉

**Comment by vshcherb** [comment]
@sonora what do you mean by automated legend? Legend plugin?

**Comment by sonora** [comment]
Yes, the Legends plugin, which produces a legend simply based on 'live-evaluating' object appearance for the current renderer. See e.g. foldout screenshot in https://github.com/osmandapp/OsmAnd/issues/20357#issuecomment-2307245131 , where it was very useful.
